gtsfm/retriever/joint_netvlad_sequential_retriever.py

```diff
@@ -0,0 +1,62 @@
+"""Retriever that includes both sequential and retrieval links.
+
+Authors: John Lambert
+"""
+
+from typing import List, Tuple
+
+import dask
+
+import gtsfm.utils.logger as logger_utils
+from gtsfm.loader.loader_base import LoaderBase
+from gtsfm.retriever.retriever_base import RetrieverBase
+from gtsfm.retriever.netvlad_retriever import NetVLADRetriever
+from gtsfm.retriever.sequential_retriever import SequentialRetriever
+
+
+logger = logger_utils.get_logger()
+
+
+class JointNetVLADSequentialRetriever(RetrieverBase):
```

Comment:
High-level design question:

Can we bake in this retrieving logic into whatever calls this class? Ideally it would be nice to specify a list of retrievers in the config and an aggregation method - union, intersection etc. and get them aggregated. Or we could create an aggregator that accepts a list of retrievers. 

Can also be done in another PR. Please create an issue maybe?